### ISSUE TITLE
//:protobuf_python has empty __init__.py

### ISSUE THREAD

**Issue by aj-michael** [google/protobuf#2833]
TLDR: I would like to add `py_extra_srcs = glob(["python/**/__init__.py"])` to `//:protobuf_python` so that I can depend on both `google.protobuf` and `google.apputils`.

I am working on a Bazel python project and would like to make use of `//:protobuf_python` from this repository.

In my `WORKSPACE` file I have

```
http_archive(
    name = "protobuf",
    url = "http://github.com/google/protobuf/archive/008b5a228b37c054f46ba478ccafa5e855cb16db.tar.gz",
    sha256 = "2737ad055eb8a9bc63ed068e32c4ea280b62d8236578cb4d4120eb5543f759ab",
    strip_prefix = "protobuf-008b5a228b37c054f46ba478ccafa5e855cb16db",
)
new_http_archive(
    name = "six_archive",
    build_file = "six.BUILD",
    sha256 = "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a",
    url = "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz#md5=34eed507548117b2ab523ab14b2f8b55",
)
bind(
    name = "six",
    actual = "@six_archive//:six",
)
```

In my `BUILD` file I have

```
py_binary(
    name = "foo",
    srcs = ["foo.py"],
    main = "foo.py",
    deps = ["@protobuf//:protobuf_python"],
)
```

In my source code I have

```
from google.protobuf import text_format
```

After running `bazel build //:foo`, `bazel-bin/foo.runfiles/protobuf/python/google/__init__.py` is empty. I would like it to contain the contents of https://github.com/google/protobuf/blob/master/python/google/__init__.py. It is empty because Bazel creates an empty `__init__.py` if one is not provided as a `src` to the `python_library`. Since the `__init__.py` is blank, it means that I cannot depend on another python library that contributes to the `google` module. In my particular case, I would like to depend on both this library and https://pypi.python.org/pypi/google-apputils.

If I use this patch, the `__init__.py` in the Bazel runfiles has the correct contents.

```
diff --git a/BUILD b/BUILD
index bf894628..d63c6d4f 100644
--- a/BUILD
+++ b/BUILD
@@ -693,6 +693,7 @@ py_proto_library(
         ":python_srcs",
         "//external:six",
     ],
+    py_extra_srcs = glob(["python/**/__init__.py"]),
     srcs_version = "PY2AND3",
     visibility = ["//visibility:public"],
 )
```
